app/code/Magento/Catalog/Block/Product/View/Attributes.php

```diff
@@ -85,8 +86,8 @@ public function getAdditionalData(array $excludeAttr = [])
                 } elseif ($attribute->getFrontendInput() == 'price' && is_string($value)) {
                     $value = $this->priceCurrency->convertAndFormat($value);
                 }
-
-                if (is_string($value) && strlen($value)) {
+    
+                if ($value instanceof Phrase || (is_string($value) && strlen($value))) {
```

Comment:
Sorry, from second look I have a little suggestion :) As `Phrase` accepts empty text, we need to check its length too.

So, it would be nice to add something like 
```
$value = $value instanceof Phrase ? (string)$value : $value;
```
prior to `if` instead of additional condition.
___
Would be nice to add a unit test for this case, but it's up to you, I believe.
___
And please link commit to your GitHub account: https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/ ;)